src/ctap/storage.rs

```diff
@@ -570,6 +570,42 @@ impl PersistentStore {
         Ok(self.store.insert(key::AAGUID, aaguid)?)
     }
 
+    /// Returns the upgrade public key if defined for the identifier.
+    // TODO remove allow
+    #[allow(dead_code)]
+    pub fn upgrade_public_key(
```

Comment:
Can't we prefix the functions with `_` and remove it at first usage? It's easier not to forget because it's local information (in the name). This is more fail-proof because otherwise we might use them and forget to remove the `allow(dead_code)` and later when we stop using the function accidentally or deliberately we are not notified.

Reply:
Done, even though it is slightly more work, as the `allow(dead_code)` is transitive for all subfunctions that now also need to get underscored.

Reply:
Oh sorry, I forgot that this was recursive :-( If you were confident you would not forget to remove the `allow(dead_code)` because it's already part of a future commit we could have temporarily kept them.